local-cli/runIOS/findMatchingSimulator.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.*
+ */
+'use strict';
+
+function findMatchingSimulator(simulators, simulatorName) {
```

Comment:
Please add a comment explaining what the function does, what the arguments are and an example way someone would call it (including example arguments).